stream-bot/bot.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Created by danielfirsht on 3/20/17.
+ */
+
+'use strict';
+
+let csync = require('csync');
```

Comment:
also, just FYI, I recently figured this out since I had just assumed javascript's let was like swift's let, but swift's let equivalent in javascript is const. you don't have to change that her, but just in case you didn't know
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.qot25hhzv